specificatie/zoek-personen.yaml

```diff
@@ -0,0 +1,333 @@
+openapi: 3.0.3
+info:
+  title: Zoek Personen endpoint definitie
+  version: 2.0.0
+  contact: {}
+paths:
+  /personen:
+    post:
+      operationId: GetPersonen
+      summary: Vindt personen
+      description: |
+        Zoek personen met één van de onderstaande verplichte combinaties van parameters en vul ze evt. aan met optionele parameters.
+
+
+        Default krijg je personen terug die nog in leven zijn, tenzij je de inclusiefoverledenpersonen=true opgeeft.
+
+
+        Gebruik de fields parameter als je alleen specifieke velden in het antwoord wil zien, [zie functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/blob/v1.3.0/features/fields_extensie.feature)
+
+
+        1.  Zoek met geslachtsnaam en geboortedatum
+            - geboortedatum
+            - geslachtsnaam (minimaal 2 karakters, [wildcard toegestaan](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature))
+
+
+        2.  Zoek met geslachtsnaam en gemeente van inschrijving
+            - gemeenteVanInschrijving
+            - geslachtsnaam (minimaal 2 karakters, [wildcard toegestaan](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/wildcard.feature))
```

Comment:
hier is ook voornamen aan toegevoegd als verplichte parameter

Reply:
done